cmd/keytransparency-client/grpcc/grpc_client.go

```diff
@@ -83,14 +83,18 @@ type Client struct {
 	cli        spb.KeyTransparencyServiceClient
 	vrf        vrf.PublicKey
 	kt         *kt.Verifier
-	log        client.VerifyingLogClient
+	log        client.LogVerifier
 	mutator    mutator.Mutator
 	RetryCount int
 	RetryDelay time.Duration
 }
 
 // New creates a new client.
-func New(mapID int64, client spb.KeyTransparencyServiceClient, vrf vrf.PublicKey, verifier crypto.PublicKey, log client.VerifyingLogClient) *Client {
+func New(mapID int64,
+	client spb.KeyTransparencyServiceClient,
```

Comment:
Good question. I started doing this when the line got unwieldy long.
The Go style guide doesn't have a strict recommendation on when to wrap lines: 
https://github.com/golang/go/wiki/CodeReviewComments#line-length 

Perhaps the inconsistency is ok, or I could go back and try to group the parameters?